.github/workflows/rust.yml

```diff
@@ -0,0 +1,22 @@
+name: Rust
+
+on:
+  push:
```

Comment:
@escanda we *never* push to main since it is a protected branch so you can remove this rule.

Reply:
although we don´t push to master, we *merge* so not sure if needed :+1: